Pages/Index.razor

```diff
@@ -22,7 +22,7 @@
 @code {
     private async Task GoToAdmission()
     {
-        NavigationManager.NavigateTo("/admission");
+        NavigationManager.NavigateTo("/akiss/admission");

```

Comment:
NavigateTo("admission")

Reply:
Akiss qoymasam 'not found' url'ni topolmay